ownCloudAppShared/Branding/Branding+App.swift

```diff
@@ -45,7 +45,8 @@ extension Branding : BrandingInitialization {
 				.sendFeedbackAddress : "ios-app@owncloud.com",
 
 				.canAddAccount : true,
-				.canEditAccount : true
+				.canEditAccount : true,
+				.enableReviewPrompt : true
```

Comment:
Default value for branded clients should be`false`

Reply:
I assumed that this value would be used even if the app is not branded, but now saw that this assumption was wrong. I changed it back to `NO` / `false` and added a note to the documentation for this parameter.